source/assets/stylesheets/components/_typography.scss

```diff
@@ -0,0 +1,18 @@
+.emperor, %emperor {
```

Comment:
BTW have you seen this changes?

I translated these ones:
* `vador` :fr:  into `vader` :gb: 
* `acolyte` :fr:  into `sidekick` :gb: 

And added the `human` for `h6`.

You might wanna update your styles & blog post.